ui/src/pages/backoffice/DocumentRequest/DocumentRequestSearch/DocumentRequestListEntry.js

```diff
@@ -0,0 +1,103 @@
+import { DocumentRequestIcon } from '@pages/backoffice';
+import { BackOfficeRoutes } from '@routes/urls';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Grid, Header, Item, List } from 'semantic-ui-react';
+
+export class DocumentRequestListEntry extends Component {
+  render() {
+    const { documentRequest } = this.props;
+
+    return (
+      <Item>
+        <Item.Content>
+          <Item.Header
+            as={Link}
+            to={BackOfficeRoutes.documentRequestDetailsFor(
+              documentRequest.metadata.pid
+            )}
+            data-test={`navigate-${documentRequest.metadata.pid}`}
+          >
+            <DocumentRequestIcon /> Literature request #
+            {documentRequest.metadata.pid}
+          </Item.Header>
+          <Grid columns={5}>
```

Comment:
It says `5` columns but I count only `3`

Reply:
I will fix it in the next PR